examples/parser/serialized_grammar/grammar.js

```diff
@@ -0,0 +1,154 @@
+var chevrotain = require("chevrotain")
```

Comment:
Also the vast majority of this example code is the parser itself, perhaps we should use a micro grammar
so the end users reading this example could focus on the thing being demonstrated more easily.